panic: shorthand redefinition on restic restore

I'm running `restic restore <snapshotsId from restic snapshots>`

## Output of `restic version`
restic version
restic 0.2.0
compiled at 2016-09-29 09:50:51 with go1.6.3 on linux/amd64
build from 3114d41cb757

## Expected behavior

I expect restic to restore my backup 

## Actual behavior
````
# restic restore aa1b779d --target /tmp
panic: shorthand redefinition

goroutine 1 [running]:
panic(0x91d0c0, 0xc82000faa0)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/spf13/pflag.(*FlagSet).AddFlag(0xc82009cbd0, 0xc8201600a0)
	/tmp/restic-build-672564917/src/github.com/spf13/pflag/flag.go:673 +0xcae
github.com/spf13/cobra.(*Command).mergePersistentFlags.func2.2(0xc8201600a0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:1247 +0x7d
github.com/spf13/pflag.(*FlagSet).VisitAll(0xc82009cd80, 0xc82015d538)
	/tmp/restic-build-672564917/src/github.com/spf13/pflag/flag.go:236 +0x85
github.com/spf13/cobra.(*Command).mergePersistentFlags.func2(0xe10de0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:1249 +0xbd
github.com/spf13/cobra.(*Command).mergePersistentFlags.func2(0xe10560)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:1252 +0xed
github.com/spf13/cobra.(*Command).mergePersistentFlags(0xe10560)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:1256 +0x297
github.com/spf13/cobra.stripFlags(0xc820019e90, 0x3, 0x3, 0xe10560, 0x0, 0x0, 0x0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:367 +0x8b
github.com/spf13/cobra.(*Command).Find.func1(0xe10560, 0xc820019e90, 0x3, 0x3, 0x7, 0x0, 0x0, 0x0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:427 +0x7d
github.com/spf13/cobra.(*Command).Find.func1(0xe10de0, 0xc82000a060, 0x4, 0x4, 0x20, 0x0, 0x0, 0x0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:435 +0x2b5
github.com/spf13/cobra.(*Command).Find(0xe10de0, 0xc82000a060, 0x4, 0x4, 0xc82000fa20, 0x0, 0x0, 0x0, 0x0, 0x0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:457 +0x183
github.com/spf13/cobra.(*Command).ExecuteC(0xe10de0, 0xc82015def0, 0x0, 0x0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:677 +0x1e6
github.com/spf13/cobra.(*Command).Execute(0xe10de0, 0x0, 0x0)
	/tmp/restic-build-672564917/src/github.com/spf13/cobra/command.go:648 +0x2d
main.main()
	/tmp/restic-build-672564917/src/cmds/restic/main.go:41 +0x147
````
btw: i'm able to restore bachups via mount.

--- a/src/cmds/restic/cmd_backup.go
+++ b/src/cmds/restic/cmd_backup.go
@@ -56,12 +56,12 @@ func init() {
 	f := cmdBackup.Flags()
 	f.StringVar(&backupOptions.Parent, "parent", "", "use this parent snapshot (default: last snapshot in the repo that has the same target files/directories)")
 	f.BoolVarP(&backupOptions.Force, "force", "f", false, `force re-reading the target files/directories. Overrides the "parent" flag`)
-	f.StringSliceVarP(&backupOptions.Excludes, "exclude", "e", []string{}, "exclude a pattern (can be specified multiple times)")
+	f.StringSliceVarP(&backupOptions.Excludes, "exclude", "e", []string{}, "exclude a `pattern` (can be specified multiple times)")
 	f.StringVar(&backupOptions.ExcludeFile, "exclude-file", "", "read exclude patterns from a file")
 	f.BoolVarP(&backupOptions.ExcludeOtherFS, "one-file-system", "x", false, "Exclude other file systems")
 	f.BoolVar(&backupOptions.Stdin, "stdin", false, "read backup from stdin")
 	f.StringVar(&backupOptions.StdinFilename, "stdin-filename", "", "file name to use when reading from stdin")
-	f.StringSliceVar(&backupOptions.Tags, "tag", []string{}, "add a tag for the new snapshot (can be specified multiple times)")
+	f.StringSliceVar(&backupOptions.Tags, "tag", []string{}, "add a `tag` for the new snapshot (can be specified multiple times)")
 }
 
 func newScanProgress(gopts GlobalOptions) *restic.Progress {

--- a/src/cmds/restic/cmd_forget.go
+++ b/src/cmds/restic/cmd_forget.go
@@ -45,16 +45,16 @@ func init() {
 	cmdRoot.AddCommand(cmdForget)
 
 	f := cmdForget.Flags()
-	f.IntVarP(&forgetOptions.Last, "keep-last", "l", 0, "keep the last n snapshots")
-	f.IntVarP(&forgetOptions.Hourly, "keep-hourly", "H", 0, "keep the last n hourly snapshots")
-	f.IntVarP(&forgetOptions.Daily, "keep-daily", "d", 0, "keep the last n daily snapshots")
-	f.IntVarP(&forgetOptions.Weekly, "keep-weekly", "w", 0, "keep the last n weekly snapshots")
-	f.IntVarP(&forgetOptions.Monthly, "keep-monthly", "m", 0, "keep the last n monthly snapshots")
-	f.IntVarP(&forgetOptions.Yearly, "keep-yearly", "y", 0, "keep the last n yearly snapshots")
+	f.IntVarP(&forgetOptions.Last, "keep-last", "l", 0, "keep the last `n` snapshots")
+	f.IntVarP(&forgetOptions.Hourly, "keep-hourly", "H", 0, "keep the last `n` hourly snapshots")
+	f.IntVarP(&forgetOptions.Daily, "keep-daily", "d", 0, "keep the last `n` daily snapshots")
+	f.IntVarP(&forgetOptions.Weekly, "keep-weekly", "w", 0, "keep the last `n` weekly snapshots")
+	f.IntVarP(&forgetOptions.Monthly, "keep-monthly", "m", 0, "keep the last `n` monthly snapshots")
+	f.IntVarP(&forgetOptions.Yearly, "keep-yearly", "y", 0, "keep the last `n` yearly snapshots")
 
-	f.StringSliceVar(&forgetOptions.KeepTags, "keep-tag", []string{}, "always keep snapshots with this tag (can be specified multiple times)")
+	f.StringSliceVar(&forgetOptions.KeepTags, "keep-tag", []string{}, "always keep snapshots with this `tag` (can be specified multiple times)")
 	f.StringVar(&forgetOptions.Hostname, "hostname", "", "only forget snapshots for the given hostname")
-	f.StringSliceVar(&forgetOptions.Tags, "tag", []string{}, "only forget snapshots with the tag (can be specified multiple times)")
+	f.StringSliceVar(&forgetOptions.Tags, "tag", []string{}, "only forget snapshots with the `tag` (can be specified multiple times)")
 
 	f.BoolVarP(&forgetOptions.DryRun, "dry-run", "n", false, "do not delete anything, just print what would be done")
 }

--- a/src/cmds/restic/cmd_restore.go
+++ b/src/cmds/restic/cmd_restore.go
@@ -39,12 +39,12 @@ func init() {
 	cmdRoot.AddCommand(cmdRestore)
 
 	flags := cmdRestore.Flags()
-	flags.StringSliceVarP(&restoreOptions.Exclude, "exclude", "e", nil, "exclude a pattern (can be specified multiple times)")
-	flags.StringSliceVarP(&restoreOptions.Include, "include", "i", nil, "include a pattern, exclude everything else (can be specified multiple times)")
+	flags.StringSliceVarP(&restoreOptions.Exclude, "exclude", "e", nil, "exclude a `pattern` (can be specified multiple times)")
+	flags.StringSliceVarP(&restoreOptions.Include, "include", "i", nil, "include a `pattern`, exclude everything else (can be specified multiple times)")
 	flags.StringVarP(&restoreOptions.Target, "target", "t", "", "directory to extract data to")
 
 	flags.StringVarP(&restoreOptions.Host, "host", "H", "", `only consider snapshots for this host when the snapshot ID is "latest"`)
-	flags.StringSliceVar(&restoreOptions.Paths, "path", nil, `only consider snapshots which include this (absolute) path for snapshot ID "latest"`)
+	flags.StringSliceVar(&restoreOptions.Paths, "path", nil, "only consider snapshots which include this (absolute) `path` for snapshot ID \"latest\"")
 }
 
 func runRestore(opts RestoreOptions, gopts GlobalOptions, args []string) error {

--- a/src/cmds/restic/cmd_restore.go
+++ b/src/cmds/restic/cmd_restore.go
@@ -43,8 +43,8 @@ func init() {
 	flags.StringSliceVarP(&restoreOptions.Include, "include", "i", nil, "include a pattern, exclude everything else (can be specified multiple times)")
 	flags.StringVarP(&restoreOptions.Target, "target", "t", "", "directory to extract data to")
 
-	flags.StringVarP(&restoreOptions.Host, "host", "h", "", `only consider snapshots for this host when the snapshot ID is "latest"`)
-	flags.StringSliceVarP(&restoreOptions.Paths, "path", "p", nil, `only consider snapshots which include this (absolute) path for snapshot ID "latest"`)
+	flags.StringVarP(&restoreOptions.Host, "host", "H", "", `only consider snapshots for this host when the snapshot ID is "latest"`)
+	flags.StringSliceVar(&restoreOptions.Paths, "path", nil, `only consider snapshots which include this (absolute) path for snapshot ID "latest"`)
 }
 
 func runRestore(opts RestoreOptions, gopts GlobalOptions, args []string) error {

--- a/src/cmds/restic/cmd_snapshots.go
+++ b/src/cmds/restic/cmd_snapshots.go
@@ -35,7 +35,7 @@ func init() {
 
 	f := cmdSnapshots.Flags()
 	f.StringVar(&snapshotOptions.Host, "host", "", "only print snapshots for this host")
-	f.StringSliceVar(&snapshotOptions.Paths, "path", []string{}, "only print snapshots for this path (can be specified multiple times)")
+	f.StringSliceVar(&snapshotOptions.Paths, "path", []string{}, "only print snapshots for this `path` (can be specified multiple times)")
 }
 
 func runSnapshots(opts SnapshotOptions, gopts GlobalOptions, args []string) error {

